### PR TITLE
Add color label new prop and minor refactor

### LINKED ISSUE #102: Feature Request: Font Colours
It would be nice if we could also change the font colour of the text for states, too.

### PR DESCRIPTION
**Pull request by guastallaigor**
This closes #102 .

This PR adds a new prop called `colorLabel` which is basically the same as the prop `color`, but it changes the color of the label instead of the background.
Since it's using the same logic (computed properties) as the `color` prop, I decided to create a new component specific for labels called `ButtonLabel.vue` (since the other component is called `Button.vue`), and to reuse the logic I created a mixin that is being used by both components. That way all the computed properties and some props are being reused and there isn't no code duplication.
I had to just remove the `consts` and changed to `data` properties, so that the colors are different in between those two components.
For me to reuse those computed properties in a mixin between those two components, the prop color being used differently in each component.
The prop color inside `ButtonLabel.vue` is being used for the label colors (font color).
The prop color inside `Button.vue` is being used for the background color of the toggle.
The `v-switch-label` class was moved to the children component.
The slots was removed for now since there wasn't no used to them.
I also removed an `isBoolean` import that was not being used.
I tested in my browser by importing the dist into another Vue project. 
The `README.md` was also updated.

-----------------------

This is the best I could do for this to be as cleanest as possible, and to have no code duplication.

image
image

Please let me know what do you think of this. 
Thank you.

### CODE REVIEW

File: src/Button.vue [hunk 1]
```diff
@@ -74,56 +74,40 @@ export default {
...
       type: [String, Object],
       validator (value) {
         return isString(value)
           || has(value, 'checked')
           || has(value, 'unchecked')
-          || has(value, 'disabled')
       }
     },
-    switchColor: {
+    colorLabel: {
```
Comment (euvl) [inline src/Button.vue]: Is'nt it a breaking change?

File: src/Button.vue [hunk 2]
```diff
@@ -18,39 +18,43 @@
...
+      :toggled="false"
+    />
   </template>
 </label>
 </template>
 
 <script>
-import { isString, isObject, isBoolean, has, get, translate, px } from './utils'
+import { isString, isObject, has, get, translate, px } from './utils'
+import generalMixin from './general-mixin'
```
Comment (euvl) [inline src/Button.vue]: I think we use camel case for file names

File: src/Button.vue [hunk 3]
```diff
@@ -239,30 +184,31 @@ export default {
   },
   data () {
     return {
-      toggled: !!this.value
+      toggled: !!this.value,
+      defaultColorChecked: '#75c791',
```
Comment (euvl) [inline src/Button.vue]: Not sure i understand this bit, seems you have this defined in another file aswell. Also what was wrong with having constants?

File: src/Button.vue [hunk 4]
```diff
@@ -142,7 +126,7 @@ export default {
     },
 
     buttonRadius () {
-      return this.height - this.margin * 2;
+      return this.height - this.margin * 2
```
Comment (euvl) [inline src/Button.vue]: There are a lot of changes where semicolon is gone.
It is quite hard now to unstained what are functional and what are stylistic changes.
You might want to put it back & mb make another PR wit stylistic changes, tnx

**Comment by guastallaigor** [comment]
Thank you. I will separate this into multiple PRs when I get some time.
